[IMP] Planning: auto plan and switching shifts (#12663)
odoo/documentation: content/applications/services/planning.rst

   - Employees with a user account are redirected to the :guilabel:`My Planning` view in the
     backend view of Odoo.

.. tip::

>    The **split shifts** tool allows to easily split long shifts into segments. Hover the mouse over
>    the desired shift. A :icon:`fa-scissors` (:guilabel:`scissors`) icon appears. Click it to divide
>    the shift into segments.

Reviewer: ```suggestion
   The **split shifts** tool allows to easily split a long shift into segments. To do so, hover the mouse over
   the desired shift and click the :icon:`fa-scissors` (:guilabel:`scissors`) icon.
```
- to avoid repetitions
- to avoid sentences that just mention what you see (especially when it's a simple icon) and instead group it with an actionable step)

  they have assigned to them.
- **Conflicts**: Employees or materials cannot be assigned multiple shifts at the same time.
- **Time off**: The employees’ time off is taken into account, as well as public holidays.
- **Working hours**: Are taken into account when assigning shifts to employees or materials. It is

>   not possible to use the :guilabel:`Auto Plan` feature for and employee who is working

Reviewer: ```suggestion
  not possible to use the :guilabel:`Auto Plan` feature for an employee who is working
```
